templates/columns/editorUI.html.ep

```diff
@@ -1,11 +1,11 @@
-%= stylesheet '/vendor/codemirror/theme/monokai.css'
-%= stylesheet '/vendor/codemirror/theme/rubyblue.css'
-%= stylesheet '/vendor/codemirror/theme/material-palenight.css'
-%= stylesheet '/vendor/codemirror/theme/tomorrow-night-eighties.css'
-%= stylesheet '/vendor/codemirror/lib/codemirror.css'
-%= javascript '/vendor/codemirror/lib/codemirror.js'
-%= javascript '/PGCodeMirror/PG.js'
-%= javascript '/PGCodeMirror/PGaddons.js'
+%= stylesheet '/node_modules/codemirror/theme/monokai.css'
+%= stylesheet '/node_modules/codemirror/theme/rubyblue.css'
+%= stylesheet '/node_modules/codemirror/theme/material-palenight.css'
+%= stylesheet '/node_modules/codemirror/theme/tomorrow-night-eighties.css'
+%= stylesheet '/node_modules/codemirror/lib/codemirror.css'
+%= javascript '/node_modules/codemirror/lib/codemirror.js'
+%= javascript '/js/apps/PGCodeMirror/PG.js'
+%= javascript '/js/apps/PGCodeMirror/PGaddons.js'
```

Comment:
The PGaddons.js file does not seem to exist anymore.  So this line should be deleted.  The console shows a loading error for this.